pdcupdater/handlers/modules.py

```diff
@@ -0,0 +1,68 @@
+import os
+import logging
+import errno
+import re
+
+import beanbag
+import pdcupdater.handlers
+import pdcupdater.services
+import rida
+
+
+log = logging.getLogger((__name__)
+
+def get_koji_tag(variant_info):
+    return "%s-%s-%s" % (variant_info['variant_id'], variant_info['variant_version'], variant_info['variant_release'])
+
+class NewUnreleasedVariantHandler(pdcupdater.handlers.BaseHandler):
+    """ When a new tree is created """
+
+    relevant_states = [rida.BUILD_STATES['init']]
+    valid_states = relevant_states + [rida.BUILD_STATES[s] for s in ('wait', 'failed', 'done', 'ready')]
```

Comment:
Discussion about `state` being a numerical or string enum notwithstanding, how about this:

    valid_states = list(rida.BUILD_STATES.values())